pkg/tools/clientmap/refcount_test.go

```diff
@@ -1,3 +1,5 @@
+// Copyright (c) 2020 Doc.ai and/or its affiliates.
```

Comment:
I feel this test can be simplified.

Please take a look at tests from sync.Map https://golang.org/src/sync/map_test.go

Reply:
It is the only way to add some _eventually failing on incorrect implementation_ test I see. There is no such tests actually in the `sync.Map` tests. Such tests should be very helpful for someone who will try to optimize the `RefcountMap` implementation.